config/tests/src/test/java/org/jboss/windup/config/Insider.java

```diff
@@ -0,0 +1,12 @@
+package org.jboss.windup.config;
+
+import org.ocpsoft.rewrite.context.EvaluationContext;
+
+/**
+ *
+ * @author Ondrej Zizka, ozizka at redhat.com
+ */
+public interface Insider
+{
+    public void call(GraphRewrite event, EvaluationContext evCtx);
+}
```

Comment:
(18:39:25) lincolnthree1: ozizka1: it's used only once, it's in a package that is not general and clear that it's intended for more general purpose
(18:39:50) lincolnthree1: ozizka1: and in fact, this type is useful as a general utility class overall.
(18:39:51) ozizka-FN: yes. Only once - but could be used multiple times if team finds it nice
(18:40:14) ozizka-FN: Probably.
(18:40:22) lincolnthree1: ozizka1: it should be in the API
(18:40:30) lincolnthree1: ozizka1: close to WindupRuleProvider
(18:40:37) lincolnthree1: ozizka1: and IteratingRuleProvider